activerecord/lib/active_record/connection_adapters/postgresql/schema_statements.rb

```diff
@@ -93,7 +93,7 @@ def indexes(table_name) # :nodoc:
             INNER JOIN pg_index d ON t.oid = d.indrelid
             INNER JOIN pg_class i ON d.indexrelid = i.oid
             LEFT JOIN pg_namespace n ON n.oid = i.relnamespace
-            WHERE i.relkind = 'i'
```

Comment:
:thinking: We can hardcode `IN ('i', 'I')` for all versions here since it will not break older versions AFAIK.

Reply:
I'm gonna try hardcoding the `IN` clause and running the test suite with older PG versions. I used PostgreSQL 11 to test this.